marathon-bigip-ctlr.py

```diff
@@ -793,6 +793,10 @@ def create_config_marathon(cccl, apps, snat_pool_name):
                     # normalize healthcheck protocol name to lowercase
                     if 'protocol' in hc:
                         hc['type'] = (hc['protocol']).lower()
+                        if 'http' in hc['type']:
```

Comment:
Is your intent to set the type to 'http' for both MESOS_HTTP and MESOS_HTTPS (as well as HTTP & HTTPS)?

Reply:
Yes. marathon-bigip-ctlr only supports client-side SSL, so backend health monitors need to be http.